public/js/components/AtomEdit/CustomEditors/StoryQuestionsEditor.js

```diff
@@ -25,10 +24,8 @@ export class StoryQuestionsEditor extends React.Component {
           <ManagedField fieldLocation="data.storyquestions.relatedStoryId" name="Related Tag" isRequired={true}>
             <FormFieldTagPicker/>
           </ManagedField>
-          <ManagedField fieldLocation="data.storyquestions.editorialQuestions" name="Editorial Questions">
-            <FormFieldArrayWrapper>
-              <StoryQuestionsQuestionSet onFormErrorsUpdate={this.props.onFormErrorsUpdate} />
-            </FormFieldArrayWrapper>
+          <ManagedField fieldLocation="data.storyquestions.editorialQuestions[0]" name="Editorial Questions">
```

Comment:
I think we should have a note here as they why it's `[0]` as it looks quite magical.